src/test/java/Homework23.java

```diff
@@ -0,0 +1,107 @@
+//    public void deleteAPlaylist(String email, String password)
+//    {
+//        String deletedPlayListMessage = "Deleted playlist";
+//        //accessUrlPage();
+//        provideEmailCredentials("james.patterson@testpro.io");
+//        providePasswordCredentials("te$t$tudent");
+//        loginButton();
+//        openPlaylist();
+//        clickDeletePlaylistButton();
+//        confirmDeletedPlaylistMessage();
+//        Assert.assertTrue(getDeletedPlaylistMessage().contains(deletedPlayListMessage));
+//    }
+
+//    @Test (dataProvider = "CorrectLoginProvider", dataProviderClass = BaseTest.class)
+//    public void createAPlaylist(String email, String password)
+//    {
+//        String createAPlaylist = "Created playlist ";
+//
+//        provideEmailCredentials("james.patterson@testpro.io");
+//        providePasswordCredentials("te$t$tudent");
+//        loginButton();
+//
+//        clickOnPlusSymbol();
+//    }
+    @Test
+    public void playSong()
+    {
+        LoginPage loginPage = new LoginPage(driver);
+        HomePage homePage = new HomePage(driver);
+        AllSongsPage allSongs = new AllSongsPage(driver);
+
+        loginPage.provideEmail("james.patterson@testpro.io");
+        loginPage.providePassword("te$t$tudent");
+        loginPage.clickLoginButton();
+
+        homePage.chooseAllSongsList();
+        allSongs.contextClickFirstSong();
+        allSongs.choosePlayOption();
+
+        Assert.assertTrue(allSongs.isSongPlaying());
+    }
+//
+//    @Test (dataProvider = "CorrectLoginProvider", dataProviderClass = BaseTest.class)
+//    public void countSongsInPlaylist(String email, String password)
+//    {
+//        //accessUrlPage();
+//        provideEmailCredentials("james.patterson@testpro.io");
+//        providePasswordCredentials("te$t$tudent");
+//        loginButton();
+//
+//        choosePlaylistByName("Playlist *");
+//        displayAllSongs();
+//        Assert.assertTrue(getPlaylistDetails().contains(String.valueOf(countSongsInPlaylist())));
+//    }
+//
+//
+//
+    @Test
+    public void hoverOverPlayButton()
+    {
+        LoginPage loginPage = new LoginPage(driver);
+        HomePage homePage = new HomePage(driver);
+        AllSongsPage allSongs = new AllSongsPage(driver);
+
+        loginPage.login();
+
+        homePage.chooseAllSongsList();
+        homePage.hoverPlay();
+
+        Assert.assertTrue(homePage.hoverPlay().isDisplayed());
+    }
+
+    @Test (dataProvider = "CorrectLoginProvider", dataProviderClass = BaseTest.class)
+    public void renamePlaylist(String email, String password)
+    {
+        //accessUrlPage();
+        //provideEmailCredentials("james.patterson@testpro.io");
+        //providePasswordCredentials("te$t$tudent");
+        //loginButton();
+
+        LoginPage loginPage = new LoginPage(driver);
+        loginPage.login();
+
+        //doubleClickPlaylist();
+        //enterNewPlaylistName();
+
+        //Assert.assertTrue(doesPlaylistNameExist());
```

Comment:
Please, add Assert

**One test - One Assert**

For example

![Clean_Test](https://github.com/testprojp/A47-QA-Automation-Framework/assets/104761482/a4897d49-3703-4166-b74e-d76bb33e86c2)